docs/io/unified_table_fits.rst

```diff
@@ -0,0 +1,629 @@
+.. _table_io_fits:
+
+
+FITS
+----
+
+Reading and writing tables in `FITS <https://fits.gsfc.nasa.gov/>`_ format is
+supported with ``format='fits'``. In most cases, existing FITS files should be
+automatically identified as such based on the header of the file, but if not,
+or if writing to disk, then the format should be explicitly specified.
+
+Reading
+^^^^^^^
+
+If a FITS table file contains only a single table, then it can be read in
+as shown below. In this example we use a file that is installed with astropy::
+
+    >>> from astropy.table import Table
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> chandra_events = get_pkg_data_filename('data/chandra_time.fits',
+    ...                                        package='astropy.io.fits.tests')
+    >>> t = Table.read(chandra_events)
+
+If more than one table is present in the file, you can select the HDU
+by index or by name::
+
+    >>> t = Table.read(chandra_events, hdu="EVENTS")
+
+In this case if the ``hdu`` argument is omitted, then the first table found
+will be read in and a warning will be emitted.
+
+You can also read a table from the HDUs of an in-memory FITS file. This will
+round-trip any :ref:`mixin_columns` that were written to that HDU, using the
+header information to reconstruct them::
```

Comment:
But round trip is not specific to the in-memory file case ?

Reply:
Fixed.